step/src/main/java/com/google/sps/CalendarManager.java

```diff
@@ -0,0 +1,150 @@
+
+  public static void createMatchEvent(String hostUserId, String guestUserId,
+                                      int year, int month, int day, int hour, int minute) {
+    createMatchEvent(new User(hostUserId), new User(guestUserId),
+                      year, month, day, hour, minute);
+  }
+
+  public static void createMatchEvent(User hostUser, User guestUser,
+                                      int year, int month, int day, int hour, int minute) {
+    if (!hostUser.isAuthenticated()) {
+      throw new IllegalStateException("The host user isn't authenticated. "
+                                      + "Unable to create match event.");
+    }
+
+    Event matchEvent = new MatchEventBuilder()
+      .setAttendees(hostUser, guestUser)
+      .setStartDateTime(year, month, day, hour, minute)
+      .build();
+
+    pushMatchEvent(hostUser, guestUser, matchEvent);
+  }
+
+  public static List<String> getScopes() {
+    List<String> scopes = new ArrayList<>();
+    scopes.add(CalendarScopes.CALENDAR_EVENTS); // "View and edit events on all your calendars"
+    return scopes;
+  }
+
+  // hostUser is the event owner and guestUser will receive an email invite
+  private static void pushMatchEvent(User hostUser, User guestUser, Event event) {
+    Calendar calendar = getCalendar(hostUser);
+    try {
+      String calendarId = "primary";
+      event = calendar.events().insert(calendarId, event).execute();
+      System.out.printf("Event created: %s\n", event.getHtmlLink());
+    } catch (Exception e) {
+      System.out.println("Unable to push match event.");
+      e.printStackTrace();
+    }
+  }
+
+  private static Calendar getCalendar(User user) {
+    return new Calendar.Builder(
+        new NetHttpTransport(),
+        new JacksonFactory(),
+        user.getCredential()
+      ).setApplicationName("Friend Matching Plus").build(); // TODO: not sure if app name matters
+  }
+
+}
+
+class MatchEventBuilder {
+  private Event matchEvent;
+
+  public MatchEventBuilder() {
+    matchEvent = new Event();
+  }
+
+  public MatchEventBuilder setAttendees(User hostUser, User guestUser) {
+    matchEvent.setSummary("FMP: " + hostUser.getName() + " / " + guestUser.getName());
+
+    matchEvent.setDescription(hostUser.getName() + " and " + guestUser.getName()
+                              + " have matched on Friend Matching Plus!");
+
+    matchEvent.setAttendees(Arrays.asList(new EventAttendee[] {
+      new EventAttendee().setEmail(guestUser.getEmail())
+    }));
+
+    return this;
+  }
+
+  // ZonedDateTime guide: https://www.baeldung.com/java-8-date-time-intro
+  public MatchEventBuilder setStartDateTime(int year, int month, int day, int hour, int minute) {
+    String dateTimeString = dateTimeString(year, month, day, hour, minute);
+
+    EventDateTime start = new EventDateTime()
+      .setDateTime(new DateTime(dateTimeString));
+      // .setTimeZone("America/Los_Angeles"); // TODO how to configure time zones?
```

Comment:
Again. Not really a TODO. "America/Los_Angeles" for timezone is the write format.

Reply:
This TODO is more of referring about how to configure to the user's local timezone. This line of code just serves as a reference of how to set the timezone but I haven't been able to figure out how to create an event based on the user's timezone yet.

Reply:
If you haven't figured it out then it should not be included in the PR at all.